marshmallow_utils/fields/edtfdatestring.py

```diff
@@ -66,7 +70,9 @@ class EDTFDateString(fields.Str):
     A string field which an using the EDTF Validator.
     """
 
-    def __init__(self, **kwargs):
+    def __init__(self, allow_date_and_time=True, **kwargs):
```

Comment:
do we need a new parameter? The EDTFValidator has already a list of types to check against, I would rather
1) create a new field EDTFDateTimeString which users EDTFValidator with default types
2) in EDTFDateString I would use EDTFValidator(types=[Date])
WDYT about this solution @slint ? My thinking is that we make EDFDateString more complex field instead of implementing dedicated fields with single responsibility. Number of if statements and parameters in a function or method in general increases a possibility of producing a bug